source/github-helpers/get-default-branch.ts

```diff
@@ -23,12 +24,17 @@ const getDefaultBranch = cache.function(async function (repository?: pageDetect.
 
 	if (arguments.length === 0 || JSON.stringify(repository) === JSON.stringify(getRepo())) {
 		if (pageDetect.isRepoHome()) {
-			const currentBranch = select('#branch-select-menu [data-menu-button]');
+			const currentBranch = await elementReady('[data-hotkey="w"]');
 			if (currentBranch) { // If missing, it'll default to the API call
```

Comment:
Does this ever happen? It feels like this should be an error. It's possible that I left it here as a fallback mechanism since it would break a lot of features, but maybe we should add a `console.error` after this condition just to make the error visible.

I'm focusing on this one because `if ! elementReady` is an antipattern which delays all the code after it until the `domready` event